utils/unittest/fixtures.go

```diff
@@ -2683,17 +2683,18 @@ func P2PRPCFixture(opts ...RPCFixtureOpt) *pubsub.RPC {
 }
 
 // GossipSubMessageFixture a gossipsub message fixture for the provided topic.
-func GossipSubMessageFixture(topic string) *pubsub_pb.Message {
+func GossipSubMessageFixture(topic string, from []byte) *pubsub_pb.Message {
```

Comment:
```suggestion
func GossipSubMessageFixture(topic string, from peer.ID) *pubsub_pb.Message {
```
In our codebase we mainly deal with peer ids in their `peer.ID` format than `[]byte`, hence the fixtures are better to perform the conversions internally.